tests/ttnn/unit_tests/operations/test_moreh_softmin.py

```diff
@@ -397,3 +388,91 @@ def test_softmin_backward_optional_output_tensor(shape_dim, optional_output_tens
     passing, out = comp_allclose_and_pcc(x.grad, tt_dev, rtol=rtol, atol=atol)
     logger.info(out)
     assert passing
+
+
+@pytest.mark.parametrize(
+    "shape_dim",
+    [
+        [[32, 32], 1],  # single tile
+        [[3, 32, 32 * 5], 2],  # mutiple tile with dim W
+        [[5, 6, 32, 32], 3],  # multiple cores
+        [[10, 20, 32 * 3, 32 * 5], 3],  # multiple tiles per core
+        [[32, 32], 0],  # single tile
+        [[3, 32 * 5, 32], 1],  # mutiple tile with dim H
+        [[5, 6, 32, 32], 2],  # multiple cores
+        [[10, 20, 32 * 3, 32 * 5], 2],  # multiple tiles per core
+    ],
+)
+@pytest.mark.parametrize("compute_kernel_options", compute_kernel_options, ids=compute_kernel_ids)
+def test_softmin_callback(shape_dim, compute_kernel_options, device, use_program_cache):
```

Comment:
1. Please reduce or change the `shape_dim` parameter to match the purpose of the test.

The purpose of this test is to verify the callback functionality, but when looking at `shape_dim`, it seems to be testing correct behavior across various shapes.

Since testing for different `shape_dim` is likely already covered by other tests, this appears to be an unnecessary duplicate test.

2. I also think that `compute_kernel_options` is unnecessary.

The more tests there are, the lower the chances of the operation failing. However, as the number of tests increases, developers have to wait longer for test execution, and GitHub Actions becomes slower.

Reply:
Should I limit the number of tests down to 1-3 tests for callback tests?

Reply:
I want to test all 5 factories (meaning 5 tests), is that okay?

Reply:
Are there specific reasons for wanting to create five tests? 
I believe if two tests are calling the same callback, they can be consolidated into one. 
However, if there are five different types of factories for softmin, and each test is targeting a different factory, then having five tests would make sense.